### DISCUSSION TITLE
Will it work on a DWIN T5UID1 480x320 screen?

### DISCUSSION THREAD

**Discussion by lijgame** [Desuuuu/DGUS-reloaded-Klipper#15] [Q&A]
Hi Desuuuu,

Thanks for the amazing port.
My printer come with a DWIN DMT48320C035_06W touchscreen, which is also running  on DWIN T5UID1 UI.
But the resolution is different, it's 480x320.

Will this port work for that resolution?

**Comment by Desuuuu** [comment]
No, the screen firmware will not work with different resolution touchscreens.

**Reply by lijgame** [reply]
Thanks for confirmation.

**Comment by SargFowler** [comment]
I'm not quite sure why not.
I've also got a 480x320 screen and I know it works with the software when I try out a demo project.
What needs to change to make it 480x320 from the default 480x272?
